src/tip.integration.ts

```diff
@@ -35,18 +32,31 @@ const getTipRequest = (tip: TipRequest["tip"], network: "localkusama" | "localpo
 
 const networks = ["localkusama", "localpolkadot"] as const;
 const tipSizes: TipRequest["tip"]["size"][] = ["small", "medium", "large", new BN("7"), new BN("30")];
+const commonDockerArgs =
+  "--tmp --alice --execution Native --ws-port 9945 --ws-external --rpc-external --no-prometheus --no-telemetry --rpc-cors all";
 
 describe("tip", () => {
   let state: State;
+  let kusamaContainer: StartedTestContainer;
   let kusamaApi: ApiPromise;
+  let polkadotContainer: StartedTestContainer;
   let polkadotApi: ApiPromise;
 
-  beforeAll(() => {
+  beforeAll(async () => {
+    kusamaContainer = await new GenericContainer("parity/polkadot:v0.9.42")
+      .withExposedPorts({ container: 9945, host: 9901 }) // Corresponds to chain-config.ts
+      .withWaitStrategy(Wait.forListeningPorts())
+      .withCommand(("--chain kusama-dev --force-kusama " + commonDockerArgs).split(" "))
```

Comment:
Maybe make it an array from the get-go? 

```ts
const commonDockerArgs = [
  "--tmp", "--alice", "--execution", "Native", "--ws-port", "9945", "--ws-external", "--rpc-external", "--no-prometheus", "--no-telemetry", "--rpc-cors", "all"];

// ...

kusamaContainer = await new GenericContainer("parity/polkadot:v0.9.42")
      .withExposedPorts({ container: 9945, host: 9901 }) // Corresponds to chain-config.ts
      .withWaitStrategy(Wait.forListeningPorts())
      .withCommand((["--chain", "kusama-dev", "--force-kusama", ...commonDockerArgs]))
```

Reply:
I made it that way so it is easily copy-able from/to a local terminal, in case you want to run the thing locally and poke around it.

Reply:
I can relate to that :)